components/tooltip/script.js

```diff
@@ -75,7 +89,10 @@ class Tooltip extends HTMLElement {
             left: elementSize.left + (elementSize.width / 2) - tooltipSize.width / 2
         }

```

Comment:
Can't we rename this variable as elementPosition, as it is very similar to this.position?

Reply:
Yes - changed.